### PR TITLE
Add tests to support zksync

### PR DESCRIPTION
**Pull request by jferas**
This PR also adds the 'initializer' modifier back on the hats module 'setup' function.

### CODE REVIEW

File: test/HatsModule.t.sol [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  string public MODULE_VERSION = "test module";
+  address hatAddress = makeAddr("this is a hat address");
+  uint256 hatId = 1;
+  HatsModule public hatsModule;
+  bytes largeBytes =
+    abi.encodePacked("This is a fairly large bytes object, but is it really? Yes it is, it's quite large");
+  uint256 public largeBytesLength = largeBytes.length;
+
+  function _deployModule(string memory _moduleVersion, address _hatAddress, uint256 _hatId) public virtual {
+    // super.setUp();
```
Comment (alexkeating) [inline test/HatsModule.t.sol]: This can probably be commented out.

File: src/HatsModule.sol [hunk 2]
```diff
@@ -38,7 +38,7 @@ contract HatsModule is IHatsModule {
   //////////////////////////////////////////////////////////////*/
 
   /// @inheritdoc IHatsModule
-  function setUp(bytes calldata _initData) public {
+  function setUp(bytes calldata _initData) public initializer {
```
Comment (alexkeating) [inline src/HatsModule.sol:41]: It is probably a good idea to add a test for this

File: test/HatsModule.t.sol (hunk 1 above)
Comment (jferas) [reply test/HatsModule.t.sol]: Done: https://github.com/alexkeating/hats-module/pull/3/commits/9caaf8ba495f4bee16257e8b0d82f808dcb7d229

File: src/HatsModule.sol (hunk 2 above)
Comment (jferas) [reply src/HatsModule.sol:41]: Done: https://github.com/alexkeating/hats-module/pull/3/commits/2f7a6451ef4c14f8e98ed925668ce098a9a03022

**Pull request merged**